spec/function.dd

```diff
@@ -1362,8 +1362,8 @@ ref int gun(return ref int x) {
 }
 ---
 
-        $(P Ref methods marked with the `return` attribute ensure the returned
-        reference will not outlive the respective aggregate instance.
+        $(P Struct non-static methods marked with the `return` attribute ensure the returned
+        reference will not outlive the struct instance.
```

Comment:
Classes?

Reply:
Interfaces?

Reply:
Classes are part of the `return scope` thing.

Reply:
If its going to confuse us, chances are it'll confuse someone else.